configs/tiny/OLMo-300M.yaml

```diff
@@ -47,47 +47,53 @@ optimizer:
   name: adamw
   learning_rate: 6.0e-4
   weight_decay: 0.1
+  decay_norm_and_bias: true
+  decay_embeddings: false
   betas:
   - 0.9
   - 0.95
   metrics_log_interval: 10
 
 scheduler:
   name: cosine_with_warmup
-  t_warmup: 2000
+  units: tokens
+  t_warmup: 4194304000
+  t_max: 3e12
   alpha_f: 0.1
+  grad_clip_warmup_steps: null
+  grad_clip_warmup_factor: 5
+  warmup_min_lr: 0
 
 tokenizer:
-  identifier: tokenizers/allenai_eleuther-ai-gpt-neox-20b-pii-special.json
+  identifier: tokenizers/allenai_gpt-neox-olmo-dolma-v1_5.json
   truncate_direction: right
 
 save_folder: workspace/${run_name}  # doesn't matter since we'll upload to S3
-remote_save_folder: s3://allennlp-ananyaj/olmo-tiny/300M/${run_name}
+remote_save_folder: s3://ai2-llm/checkpoints/olmo-tiny/300M/${run_name}
 save_overwrite: false
-# Sharded checkpoints (best for restarts)
-save_interval: 5000
-save_num_checkpoints_to_keep: 3
+
 # Unsharded checkpoints (for ddp)
 save_interval_unsharded: 5000
-save_num_unsharded_checkpoints_to_keep: 3
+save_num_unsharded_checkpoints_to_keep: -1
 
 load_path: null
 
-max_duration: 100_000  # 419B tokens, this is for the scheduler
-stop_at: 100_000
-global_train_batch_size: 2048
-device_train_microbatch_size: 8
+max_duration: 2ep
```

Comment:
This means you'll run into this bug: https://github.com/allenai/OLMo/issues/584
It might not matter. The problem is only that the second epoch will be shuffled the same way the first one is shuffled.

Reply:
I'll add a ``stop_at`` 400k steps!